.github/workflows/CI.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.0'
+          - 'lts'
           - '1'
           - 'nightly'
```

Comment:
IMO it's not very useful to test nightly. Maybe we could test `pre` but I'm not sure if it's worth it.
```suggestion
```